.github/workflows/test.yml

```diff
@@ -160,9 +167,31 @@ jobs:
           else
             echo "value=$PATH" >> $GITHUB_OUTPUT
           fi
+          if [[ '${{ inputs.static-suffix }}' == '-static' ]]; then
+            echo "static-problemlist-path=`pwd`/test/hotspot/jtreg/ProblemList-StaticJdk.txt" >> $GITHUB_OUTPUT
+          fi
+
+      - name: 'Run tests on static JDK'
+        id: run-tests-static
+        if: ${{ inputs.static-suffix == '-static' }}
+        run: >
+          make test-prebuilt
+          TEST='${{ matrix.test-suite }}'
+          BOOT_JDK=${{ steps.bootjdk.outputs.path }}
+          JDK_UNDER_TEST=${{ steps.bundles.outputs.static-jdk-path }}
+          JDK_FOR_COMPILE=${{ steps.bundles.outputs.jdk-path }}
```

Comment:
Do we really need to duplicate all this code? From what I can see, this is just to be able to send in the JDK_FOR_COMPILE argument, right?

Reply:
> Do we really need to duplicate all this code? From what I can see, this is just to be able to send in the JDK_FOR_COMPILE argument, right?

Yaml syntax is new to me. I went with that as it worked. :-) The static case sets the `JDK_FOR_COMPILE`|`JDK_UNDER_TEST` and `EXTRA_PROBLEM_LISTS` in `JTREG`. Any suggestion?

Reply:
I think you could add something like 
```
${{ inputs.extra-test-options }}
```
in the `make test-prebuilt` command line, and then set it up as arguments when calling the test workflow.

Reply:
Did you try this?

Reply:
I missed your above suggestion earlier. Experimenting with this. Thanks for the suggestion.

Reply:
I went with using an explicit `extra-options` step to set up the extra command-line options for `make test-prebuilt` to handle the static case. That got rid of the separate `run-tests-static` step and notify error step. The `run-tests` now is used for running tests for both the regular and static cases. GHA testing showed that it worked ok.